src/less/less/mixins-common.less

```diff
@@ -1,13 +1,19 @@
 @import "variables-common.less";
 
 // base styles for BEM span.icon block
-.background-icon-base() {
+.background-icon-image-base() {
     background-repeat: no-repeat;
-    background-size: contain;
     display: inline-block;
     vertical-align: middle;
 }
 
+// DEPRECATED - do not use
+.background-icon-base() {
+    .background-icon-image-base();
+
+    background-size: contain;
+}
+
 // base styles for BEM svg.icon block
 .foreground-icon-base() {
```

Comment:
Should we also create a new alias for foreground icon to match? i.e.`foreground-icon-image-base()` Or not worth the hassle (considering we will be hiding these away in Skin v8)?

Reply:
Thoughts on the best way to "hide" it?

Reply:
Yeah, let's wait.